src/ogdf/cluster/HananiTutteCPlanarity.cpp

```diff
@@ -325,22 +325,12 @@ class HananiTutteCPlanarity::CGraph {
 
 	CLinearSystem m_ls;
 
-	int64_t m_tPrepare;
-	int64_t m_tCreateSparse;
-	int64_t m_tSolve;
-
-
 public:
 	explicit CGraph(const ClusterGraph& C);
+	~CGraph() override = default;
 
-	bool cplanar(int& nRows, int& nCols);
-	Verification cpcheck(int& nRows, int& nCols);
-
-	int64_t timePrepare() const { return m_tPrepare; }
-
-	int64_t timeCreateSparse() const { return m_tCreateSparse; }
-
-	int64_t timesolve() const { return m_tSolve; }
```

Comment:
Since these are public, it would be best to mention these changes in the porting guide `unreleased.md`, i.e. that `HananiTutteCPlanarity::cplanar()` was renamed to `test()` and `HananiTutteCPlanarity::cpcheck()` to `verify()` (now with return value bool).

Reply:
The methods are public, but the class is only defined in this .cpp file, so its pretty inaccessible without making weird changes to the OGDF build. The parent class HananiTutteSolver from the header file was added by me to make these methods accessible in the first place.

Reply:
Oh, you're right, I did not even register the weird declarations in the cpp file. So just disregard what I wrote here.